Add guide for storing sensitive information (#38)
hyphacoop/handbook: guides.md

- [Meetings](#meetings)
- [Shortlinks](#shortlinks)

> - [Storage: Sensitive Data](#storage-sensitive-data)

Reviewer: Maybe the `Storage: ` makes the index looks confusing?

```suggestion
- [Sensitive Data](#storage-sensitive-data)
```

Author: ```suggestion
- [Sensitive Data](#sensitive-data)
```

[Chrome](https://www.techrepublic.com/article/pro-tip-add-custom-search-engines-in-chrome-for-more-efficient-searching/)
| [Firefox (requires extension)](https://addons.mozilla.org/en-US/firefox/addon/add-custom-search-engine/)

> ## Storage: Sensitive Data

Reviewer: ```suggestion
## Sensitive Data
```

In general, most staff prefer not to use third-party SaaS services, like Google Drive, for storing sensitive information.

We store **short strings** (like SINs or codes) as individual credentials in Passbolt. These can be shared with both the providing individual and a privileged group that requires access, like Finance WG.

> If we must **store sensitive docs** (like PDFs) on GDrive for convenience, we first encrypt them with a password. We keep this in Passbolt as entry "GDrive: File Encryption". All employees can access this password. Simpler encryption schemes are preferred, for example, default PDF encryption.

Reviewer: Should we use generic `Drive` instead of `GDrive` in case we change the Drive solution in the future?

Author: ```suggestion
If we must **store sensitive docs** (like PDFs) in a shared drive for convenience (e.g., Google Drive),
we first encrypt them with a password.
We keep this in Passbolt under the entry "Shared Drive: File Encryption".
All employees can access this password.
Simpler encryption schemes are preferred, for example, default PDF encryption.
```

this work?

Author: Maybe we should recommend `some-file.encrypted.pdf` or `some-file.enc.pdf`

[Chrome](https://www.techrepublic.com/article/pro-tip-add-custom-search-engines-in-chrome-for-more-efficient-searching/)
| [Firefox (requires extension)](https://addons.mozilla.org/en-US/firefox/addon/add-custom-search-engine/)

## Sensitive Data

In general, most staff prefer not to use third-party SaaS services, like Google Drive, for storing sensitive information.

> We store **short strings** (like SINs or codes) as individual credentials in Passbolt. These can be shared with both the providing individual and a privileged group that requires access, like Finance WG.

Author: Re-read made it seem confusing. Leaning on passbolt terminology.

```suggestion
We store **short strings** (like SINs or codes) as password entries in Passbolt. These entries should be shared with both a privileged _group_ that needs access (e.g., Finance WG) and the _individual_.
```
